app/pages/patients/patients.js

```diff
@@ -34,13 +34,9 @@ import Invitation from '../../components/invitation';
 export let Patients = React.createClass({
   propTypes: {
     user: React.PropTypes.object,
-    fetchingUser: React.PropTypes.bool,
     patients: React.PropTypes.array,
-    fetchingPatients: React.PropTypes.bool,
     invites: React.PropTypes.array,
-    fetchingInvites: React.PropTypes.bool,
```

Comment:
Oh, now I see what the question about the super-destructuring was about! I like this. Though isn't `loading` now a prop? Add it to the propTypes please?